banzai_nres/continuum.py

```diff
@@ -3,16 +3,48 @@
 from banzai_nres.fitting import fit_polynomial
 import numpy as np
 
+from scipy.ndimage.morphology import binary_dilation
+from scipy.stats import median_absolute_deviation
+
+# Wavelength regions where there are strong Balmer or other absorption lines. This is from CERES (thanks!)
```

Comment:
Consider putting more explicit citation to CERES (i.e., to the paper and/or GitHub)